io-engine/tests/lvs_pool.rs

```diff
@@ -381,4 +445,6 @@ async fn lvs_pool_test() {
     .await;
 
     common::delete_file(&[DISKNAME2.into()]);
+    common::detach_loopdev(ldev.as_str());
```

Comment:
hmm, RAII here may be too much to do in test. w.r.t cleanup from cargo script, are you suggesting doing `losetup -D` from `clean-cargo-tests.sh` ? That might detach something we didn't attach as well. The problem with detaching loopdev is that if there is no user anymore, the detach succeeds, otherwise the device lingers with a `auto clear` flag and doesn't go away. like below:
```
NAME       SIZELIMIT OFFSET AUTOCLEAR RO BACK-FILE                    DIO LOG-SEC
/dev/loop0       0   0         1       0 /home/user/disk1              0     512
```

Reply:
Not -D, only the devices which we had created. Is it possible to add some kind of flag? Or perhaps we can place the backing files in specific directory and detach those?

> otherwise the device lingers with a auto clear flag and doesn't go away. like below:

You mean if the file gets deleted first, then loopdev gets stuck?

Reply:
> Or perhaps we can place the backing files in specific directory and detach those?

I can keep keep the backing file in `/tmp/loop_disks/<filename>` instead of directly under `/tmp`. Let the cargo test do detachments as it is doing now. Additionally, from the cleanup script do device detach for each device by getting by backing filename like `losetup -j /tmp/loop_disks/disk1.img -O NAME --noheadings`, for each backing file found under `/tmp/loop_disks/`. wdyt?

> You mean if the file gets deleted first, then loopdev gets stuck?
No, not if backing file is deleted. It is most likely an open descriptor or a mounted filesystem etc left uncleaned for the device. e.g. I had a loopdev used by lvm, and now that I deleted lvm setup the device isn't detaching.

Reply:
Yeah sounds good, maybe I'd suggest /tmp/io-engine/ to make it clear it's our stuff? Really we should do that for other stuff as well tbh

Reply:
Done.